lib/engines/content_object_store/app/views/content_object_store/content_block/editions/edit.html.erb

```diff
@@ -7,4 +7,10 @@
   } %>
 <% end %>
 
+<% if @content_block_edition.state == "scheduled" %>
```

Comment:
Minor suggestion: I think we can use enums to do `@content_block_edition.scheduled?` if we want to make this more concise and be less dependent on the actual value (so it could change).

Reply:
Good call! 👍